src/App.css

```diff
@@ -1,38 +1,38 @@
 .App {
-  text-align: center;
+	text-align: center;
+	/* background-color: #282c34; */
+	color: white;
 }
 
 .App-logo {
```

Comment:
I think a lot of this css is still from the example hello world app, so you can get rid of it now :)

Reply:
That's weird. In my version it is commented out completely. Says it was committed 2 weeks ago.